config.default.yaml

```diff
@@ -223,7 +223,9 @@ plotting:
   storage_techs: ["hydro+PHS", "battery", "H2"]
   load_carriers: ["AC load"]
   AC_carriers: ["AC line", "AC transformer"]
-  link_carriers: ["DC line", "Converter AC-DC"] 
+  link_carriers: ["DC line", "Converter AC-DC"]
+  heat_links: []
+  heat_generators: []
```

Comment:
These were removed a while ago.

Reply:
Probably we should remove
https://github.com/PyPSA/pypsa-eur/blob/036d44418da5db156741249cc7e2323788fe508c/scripts/_helpers.py#L67-L68 then.

Reply:
yes, makes totally sense